tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -597,6 +597,17 @@ TEST_P(CPUOnly, groupConvTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+/// This test targets the DKKC8 optimization.
+TEST_P(CPUOnly, nonSquarePaddingConvTest) {
+  std::array<size_t, 4> S{{1, 2, 1, 128}};
+  Tensor out1(ElemKind::FloatTy, S);
+  Tensor out2(ElemKind::FloatTy, S);
+  inferNonSquarePaddingConv(&out1, BackendKind::CPU);
```

Comment:
would be better to use backendKind_ here, this way once we add support to the OpenCL we can just change the parameterization name.

Reply:
I actually asked to do CPUOnly, because this test is specific to DKKC8 (see comment). OpenCL doesn't have DKKC8 optimization.

Reply:
maybe then reflect this in a test name and remove a comment.

Reply:
@rdzhabarov This similar function name issue had for group arg in conv before. In the same file, we have a test "TEST_P(CPUOnly, groupConvTest)", which is targeting group arg in DKKC8 mode. I used "DKKC8" in the test name before. But after discussion, since although "the test creates assumes it will be optimized into a convDKKC8, but future things could change and that may no longer occur", a more general name is used. For this test, I think it is the same :)

Reply:
that's essentially the question of what will be updated (test name or comment, either way, one should be updated if optimization does not take place). I'm fine to keep comment. Would be good to have the same behavior in the group conv test as it uses: "inferGroupConv(&out1, backendKind_);" instead of CPU directly.